### PR TITLE
0065 convenience method for location provider

### LINKED ISSUE #65: Provide convenience method for location provider if location is empty
Check in beacon.

### PR DESCRIPTION
**Pull request by Steppschuh**
Closes #65

### CODE REVIEW

**Comment by marvinmirtschin** [comment]
Increment version name and code.

File: BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/Location.java [hunk 1]
```diff
@@ -246,4 +246,9 @@ public double getElevation() {
     public void setElevation(double elevation) {
         this.elevation = elevation;
     }
+
+    public double getHeight() {
+        return Math.max(altitude, elevation);
```
[the remark this answers is not in the document]
Comment (Steppschuh) [reply BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/Location.java]: Altitude: Distance to sea level
Elevation: Distance to floor

Altitude includes the elevation.

File: BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/distance/LocationDistanceCalculator.java [hunk 2]
```diff
@@ -12,8 +12,8 @@
 
     public static double calculateDistanceBetween(Location fromLocation, Location toLocation) {
         return calculateDistanceBetween(
-                fromLocation.getLatitude(), fromLocation.getLongitude(), fromLocation.getAltitude() + fromLocation.getElevation(),
-                toLocation.getLatitude(), toLocation.getLongitude(), toLocation.getAltitude() + toLocation.getElevation()
+                fromLocation.getLatitude(), fromLocation.getLongitude(), fromLocation.getHeight(),
```
[the remark this answers is not in the document]
Comment (Steppschuh) [reply BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/distance/LocationDistanceCalculator.java]: Yes, was incorrect before

File: BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/Location.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (marvinmirtschin) [reply BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/Location.java]: Then `elevation` should only be used if on `alititude` is provided.

**Pull request merged**